types/dbt/exposure.ts

```diff
@@ -0,0 +1,14 @@
+// Reference: https://docs.getdbt.com/reference/exposure-properties
+export type DbtExposure = {
+  description?: string;
+  depends_on: { macros: [string]; nodes: [string]; sources: [string] };
+  label?: string;
+  maturity?: "high" | "medium" | "low";
+  name: string;
+  owner: { email: string; name: string };
+  tags: [string];
+  unique_id: string;
+  url?: string;
+  type: "dashboard" | "notebook" | "analysis" | "ml" | "application";
+  config: { enabled: boolean };
+};
```

Comment:
Move to domain.ts

Reply:
`domain.ts` has types specific to altimate extension types and also could not be shared with other packages in the repo. Reason for creating separate types directory is to have abstraction and ability to share between packages in repo. Also this particular file is to define types of dbt core apis, which is needed in all packages. Please let me know if you think it is still to be moved to domain.ts. @mdesmet

Reply:
I get what you are trying to do but it is premature. The other classes are actually also dbt core related data classes. Please move it to domain and also name the class similar to the other classes, ie. without Dbt prefix.

Let's first think this refactoring through instead of just applying it to one class.